oasislmf/cmd/model.py

```diff
@@ -520,59 +520,68 @@ def action(self, args):
         if not (lookup_config_fp or (keys_data_path and model_version_file_path and lookup_package_path)):
             raise OasisException('Either the lookup config JSON file path or the keys data path + model version file path + lookup package path must be provided')
 
+        # Load Location exposure files
+        source_exposures_file_path = as_path(
+            inputs.get('source_exposures_file_path', required=True, is_path=True), 'Source exposures file path'
+        )
         canonical_exposures_profile_json_path = as_path(
             inputs.get('canonical_exposures_profile_json_path', required=True, is_path=True),
             'Supplier canonical exposures profile JSON path'
         )
-        canonical_accounts_profile_json_path = as_path(
-            inputs.get('canonical_accounts_profile_json_path', required=False, is_path=True),
-            'Supplier canonical accounts profile JSON path'
+        source_to_canonical_exposures_transformation_file_path = as_path(
+            inputs.get('source_to_canonical_exposures_transformation_file_path', required=True, is_path=True),
+            'Source to canonical exposures file transformation file path'
         )
-        source_exposures_file_path = as_path(inputs.get('source_exposures_file_path', required=True, is_path=True), 'Source exposures file path')
-        source_accounts_file_path = as_path(inputs.get('source_accounts_file_path', required=False, is_path=True), 'Source accounts file path')
         source_exposures_validation_file_path = as_path(
             inputs.get('source_exposures_validation_file_path', required=False, is_path=True),
             'Source exposures validation file',
             preexists=False
```

Comment:
The use of `preexists=False` is not intuitive here - the validation files are certainly not required (as they are optional for the transformation) but they do already exist. Does it not work if you leave it out? The point of `preexists` is to indicate that a certain file path, like for an output file, is created during the process of or is the result of running a command, e.g. the keys file path.